README.md

```diff
@@ -1,14 +1,15 @@
 # Service Fabric
 
-Service Fabric is a distributed systems platform for packaging, deploying, and managing stateless and stateful distributed applications and containers at large scale. Service Fabric runs on Windows and Linux, on any cloud, any datacenter, across geographic regions, or on your laptop.
+[Service Fabric is a distributed systems platform](https://github.com/GitTorre/service-fabric/tree/master/docs/architecture/README.md) for packaging, deploying, and managing stateless and stateful distributed applications and containers at large scale. Service Fabric runs on Windows and Linux, on any cloud, any datacenter, across geographic regions, or on your laptop.
```

Comment:
I think it's kind of weird to have a hyperlink on the first couple words of a document.. Just doesn't feel quite right, like you're potentially immediately directing the user away from the document